# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: testing/web-platform/meta/fullscreen/api/element-request-fullscreen-two-elements.html.ini
# Commit: 46fd81530a8b
# Full Hash: 46fd81530a8b6d81b5520aff9451d49649f1420a
# Author: Ziran Sun <zsun@igalia.com>
# Date: 2023-05-23 09:47:12
# Regressor Bug: 1832837
# File Overlap Count: 1
# Description:
#   Bug 1832837 - Remove the fullscreen element ready check on fullscreen element descendant. r=edgar
#   
#   https://fullscreen.spec.whatwg.org/#fullscreen-element-ready-check
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D177913
# ==============================================================================

diff -r 069f4fb8fd87 -r 46fd81530a8b testing/web-platform/meta/fullscreen/api/element-request-fullscreen-two-elements.html.ini
--- a/testing/web-platform/meta/fullscreen/api/element-request-fullscreen-two-elements.html.ini	Mon May 22 11:53:20 2023 +0000
+++ b/testing/web-platform/meta/fullscreen/api/element-request-fullscreen-two-elements.html.ini	Mon May 22 12:20:47 2023 +0000
@@ -1,5 +1,3 @@
 [element-request-fullscreen-two-elements.html]
   expected:
     if (os == "android") and fission: [OK, TIMEOUT]
-  [Element#requestFullscreen() on two elements in the same document]
-    expected: FAIL